### ISSUE TITLE
Migration Procedure!

### ISSUE THREAD

**Issue by victordariovera** [firefly-iii/firefly-iii#1914]
Hi! I need to migrate Firefly between two servers. In a fresh firefly installation (over LAMP, no docker) I copied all DB data, grant privileges to user, copy .env file and folders storage/upload and storage/database... But I only achieve a "Le Right Back" message on new server.

Can you do a guide for migrating Firefly?

**Comment by victordariovera** [comment]
Hi! Can you send me the path of logs? Are app logs generated by php o you refer to php/apache standard logs?

**Comment by JC5** [comment]
They should be stored in `/storage/logs`. Check out [how to enable debug mode](https://github.com/firefly-iii/help/wiki/Enable-debug-logging-and-debug-mode).

**Comment by victordariovera** [comment]
Thanks! I activated debug logs and I could solve the issue by correcting this:

- Migrated storage folder did not have right permissions so I corrected them.
- Mysql user did not auth with password configured in .env file. I reseted them.
- When I tried to login on screen appears an error refering to "decrypt" some values. The same page indicates run "sudo php artisan cache:clear". This solved login error. 
- In app amounts do not show decimals, but a warning in page's header give me a link where indicate run this commands: "sudo apt-get install -y language-pack-en-base language-pack-nl-base", "sudo apt-get install -y language-pack-de-base language-pack-pt-base", "sudo locale-gen".

Problem solved!
